src/main/java/onlinebookstore/model/User.java

```diff
@@ -50,10 +50,9 @@ public class User implements UserDetails {
 
     private String shippingAddress;
 
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
ManyToMany has `default FetchType.LAZY;`
```suggestion
    @ManyToMany
```